README.md

```diff
@@ -1,6 +1,9 @@
 # qb-vehiclekeys
 Vehicle Keys System For QB-Core
 
+# [video]
+https://youtu.be/7E9TXR3lXPI
```

Comment:
Idea:

Maybe we should embed the video?
You can use html embed inside github .md files :)

Reply:
done thank you